src/views/Pages/Accout/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react'
+// import Dashboard from '../Dashboard'
+import Payment from '../../../components/Payment'
+import { isAuthEmail, authenticateUser, isAuthUserType } from '../../../services/Auth'
+import { Link, withRouter } from 'react-router-dom';
+import CustomerMap from '../../Map/CustomerMap'
+import {
+  Badge,
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  Col,
+  Collapse,
+  DropdownItem,
+  DropdownMenu,
+  DropdownToggle,
+  Fade,
+  Form,
+  FormGroup,
+  FormText,
+  FormFeedback,
+  Input,
```

Comment:
Codacy found an issue: ['Input' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5714105)